functions/Get-DbaDatabaseSnapshot.ps1

```diff
@@ -16,8 +16,8 @@ Credential object used to connect to the SQL Server as a different user
 .PARAMETER Databases
 Return information for only specific base dbs
 
-.PARAMETER Exclude
-Return information for all but these specific base dbs
+.PARAMETER Snapshots
```

Comment:
Yes, usually. But I tried singular and didn't like it. It didn't convey what I wanted to SQL people, which is that singular params can do arrays.